File: 10818.py
numOfCases = int(input())
nums = list(map(int, input().split()))

maximum = nums[0]
minimum = nums[0]

for each in nums:
    if each > maximum:
        maximum = each
    elif each < minimum:
        minimum = each

print("%d %d"%(minimum, maximum))

File: 10818.c
#include<stdio.h>

int main(){
    int numOfCases = 0;
    scanf("%d", &numOfCases);
    int nums[numOfCases];
    
    for(int i = 0; i < numOfCases; i++)
        scanf("%d", &nums[i]);
    
    int minimum = nums[0];
    int maximum = nums[0];
    
    for(int i = 0; i < numOfCases; i++){
        if(nums[i] > maximum)
            maximum = nums[i];
        else if(nums[i] < minimum)
            minimum = nums[i];
    }
    printf("%d %d", minimum, maximum);
    return 0;
}
